sdk/databox/Microsoft.Azure.Management.DataBox/src/Microsoft.Azure.Management.DataBox.csproj

```diff
@@ -24,6 +24,10 @@
                 4) Ordering a pickup from logistic partners for the device
                 
                 5) Fetching list of available skus in Data Box family based on customer location
+                
+                6) Validating Job creation inputs and its supportability by Azure Data Box service
```

Comment:
Are the above 5 notes from last release or for this release? If last, I believe you should remve them.

Reply:
If I understand you correctly, You want only the changes in this PR to be mentioned here